### PR TITLE
EIP-689: clarifications

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 689 requires approval from one of (@pirapira)

**Comment by axic** [comment]
@pirapira could you leave a comment if you are against these clarifications?

File: EIPS/eip-689.md [hunk 1]
```diff
@@ -38,11 +39,21 @@ It seems impractical to implement never-used features just for passing tests.  C
 
 This EIP is backwards compatible on the main network.
 
+## Acknowledgements
+
+This has been previously suggested as [EIP-684](https://github.com/ethereum/EIPs/issues/684) and [EIP-83](https://github.com/ethereum/EIPs/issues/83).
+
```
Comment (MicahZoltu) [inline EIPS/eip-689.md:45]: I'm not a fan of Acknowledgements section, I don't think it belongs in a technical specification (especially a CC0 licensed one).  I don't want this to turn into some sort of political/status thing where people cross-link to each other and give shout outs and kudos like we see in academia and at awards ceremonies.

**Comment by MicahZoltu** [comment]
@axic Was there a reason you requested a review from editors on this?  This EIP has an author, it is entirely up to them what to include.  The one exception would be a force move to `Abandoned` which I think is up to the EIP editors to decide on (not really up to the author, since they are likely not responding which is why we are moving to abandoned).

**Comment by axic** [comment]
This has a long history, was discussed on ACD, and one of the hard forks implicitly contains this EIP without mentioning it. The result of the discussion can be found on the discussions-to url.

**Comment by MicahZoltu** [comment]
> This has a long history, was discussed on ACD, and one of the hard forks implicitly contains this EIP without mentioning it. The result of the discussion can be found on the discussions-to url.

Why does that preclude the author from updating/maintaining this EIP?  Is the issue that this EIP is abandoned by the author but it should be pushed through to final despite that?  If so my preferred solution would be to find someone to champion a new EIP (it can start out as a clone of this one) and push that EIP through to final.  In the meantime, we can start the process of marking this as abandoned.

Alternatively, if we wanted to cut some corners we could just change/add an author to this EIP and that person can then push this through to final.

**Comment by axic** [comment]
There's no cutting corners here, waiting for @pirapira to respond.

**Pull request closed without merging**